helix-query.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
+indices:
+  default:
+    include:
+      - /**
+    exclude:
+      - '/drafts/**'
+      - '/tools/**'
+      - '/fragment/**'
+      - '/nav'
+      - '/footer'
```

Comment:
we would need to exclude icons folder as well

Reply:
we target folders which have documents, so this one is not required